Common/GPU/ShaderWriter.cpp

```diff
@@ -472,7 +479,11 @@ void ShaderWriter::DeclareTexture2D(const SamplerDef &def) {
 		break;
 	case GLSL_VULKAN:
 		// In the thin3d descriptor set layout, textures start at 1 in set 0. Hence the +1.
```

Comment:
Should probably s/1/texBindingBase_`.

-[Unknown]

Reply:
Fixing the comment.